src/Requests/Generator.php

```diff
@@ -47,7 +47,10 @@ public function sign(Request $request) : Request
         $key = $this->configuration->getSigningKey();
 
         $request = $request->withHeader('X-SIGNED-ID', (string) Uuid::uuid4());
-        $request = $request->withHeader('X-SIGNED-TIMESTAMP', (string) Carbon::now());
+        $request = $request->withHeader(
+            'X-SIGNED-TIMESTAMP',
+            Carbon::now()->format('Y-m-d H:i:s')
```

Comment:
Since we're looking to consume in this format I felt like it made sense to start ensuring we produce timestamps in this format.

Reply:
Note this is the default format produced when you cast `Carbon::now()` to a string so this is equivalent to what it was before.